### PR TITLE
Add dark mode support for input text

### PR DESCRIPTION
**Pull request by dannycolin**
This patch adds dark theme support for the input text of the comment form. I tried to reused the colors already present in `style.css` and only added a new one for the background-color.

Before:

image

After:

image

### CODE REVIEW

**Comment by MelissaAutumn** [comment]
Hey @dannycolin can you provide a small description / image of the change. I'm also tagging someone from design to check over the colour usage. 😄

File: style.css [hunk 1]
```diff
@@ -44,6 +44,11 @@ Tags: one-column, accessibility-ready, custom-menu, featured-images, footer-widg
 	--content-width:         1080px;
 	--featured-height:       450px;
 	--border-radius:         3px;
+
+  --form-background-color:       #ffffff;
+  --form-border-color:           #cacad8;
+  --form-border-color-focused:   #34343a;
+  --form-text-color:             #000000;
```
Comment (MelissaAutumn) [inline style.css:51]: Looks like we use tabs in style.css? 😱 

We might want to fix that later on, but unfortunately I must ask you to tab-ify all of your spaces in style.css

Comment (dannycolin) [reply style.css]: I re-add the missing evil characters :)

**Review by devmount — APPROVED** [review]

**Review by MelissaAutumn — APPROVED** [review]
Pending a quick colour check

**Review by micahilbery — APPROVED** [review]
This looks good to me and all the focus and text colors have proper contrast. :+1:

**Pull request merged**